hermes-api/src/main/java/pl/allegro/tech/hermes/api/Subscription.java

```diff
@@ -445,7 +445,7 @@ public void setModifiedAt(Long modifiedAt) {
 
     @Override
     public Subscription anonymize() {
-        if (getEndpoint().containsCredentials() || hasOAuthPolicy()) {
+        if (endpoint != null && getEndpoint().containsCredentials() || hasOAuthPolicy()) {
```

Comment:
**Nice to have**: 
Direct access to the `endpoint` field is not consistent with existing code style. I suggest to use `getEndpoint()`.

Reply:
Hi, 
I've updated the source branch to comply